src/modules/rewards/services/arb-rebate-service.ts

```diff
@@ -320,6 +322,17 @@ export class ArbRebateService {
     }
   }
 
+  private getInputTokenPrice(deposit: PartialDeposit): string {
+    if (
+      deposit.sourceChainId === ChainIds.blast &&
+      deposit.token.symbol === "USDB" &&
+      deposit.outputToken.symbol === "DAI"
```

Comment:
Will this condition ever not be true with blast? Could we consider removing the check for `deposit.outputToken.symbol === "DAI"`?

Reply:
My understanding is that at the contract level it is possible to set different input and output tokens so I think we should keep this check.